src/components/pages/NotePage.tsx

```diff
@@ -220,12 +220,18 @@ const NotePage = ({ storage }: NotePageProps) => {
 
   const { showSearchModal } = useSearchModal()
 
+  const storageTags = useMemo(() => {
+    if (storage == null) return []
+    return values(storage.tagMap).map((tag) => tag)
```

Comment:
`[...storage.tagMap.values()]`

Reply:
There is no values() on ObjectMap<T>

So final should be:
`return [...values(storage.tagMap)]`? copy of tags
or maybe even
`return values(storage.tagMap)`? raw values